src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +25,23 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        File file = new File(fileName);
+        Set<Character> tree = new TreeSet<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            String line = "";
+            while ((line = bufferedReader.readLine()) != null) {
+                for (char c : line.toLowerCase().replaceAll("[^a-z]", "").toCharArray()) {
+                    tree.add(c);
+                }
+            }
+        } catch (IOException e) {
+            throw new FileNotFoundException("File not found");
+        }
```

Comment:
you are throwing checked Exception from this method anyway and indicate this in method signature. try just removing this catch block